recipes-ccsp/ccsp/rdk-vlanmanager.bb

```diff
@@ -7,7 +7,7 @@ DEPENDS = "ccsp-common-library dbus rdk-logger utopia hal-platform libunpriv"
 
 require recipes-ccsp/ccsp/ccsp_common.inc
 
-GIT_TAG = "v1.0.0"
+GIT_TAG = "RC1.1.0a"
```

Comment:
I wonder should a release candidate change goes to a feature / sprint branch instead?